CMakeLists.txt

```diff
@@ -379,7 +385,7 @@ if (NOT ("${CMAKE_C_COMPILER_FRONTEND_VARIANT}" STREQUAL "MSVC"))
 
 	# OSX Mach-O doesn't support linking with '-Bsymbolic-functions'.
 	# Others may not support it, too.
-	list(APPEND CMAKE_REQUIRED_LIBRARIES "-Wl,-Bsymbolic-functions")
```

Comment:
Unconditionally adding `-lamiga` isn't going to work on most systems.
Adding `-lc` at all sounds like the wrong thing to do too.
If these are actually needed then it needs to be limited to just the environments where that's the case.  Also, it definitely shouldn't be lumped in with passing the `-Bsymbolic-functions` flag.

Reply:
How about adding a `list(APPEND CMAKE_REQUIRED_LIBRARIES "-Wl,-lamiga")` up in the `if (${CMAKE_SYSTEM_NAME} STREQUAL "AmigaOS")` block above?

Reply:
Oops this was added as a test and turns out it's not required at all. I've removed it.